### PR TITLE
Use ICD loader on OSX

### PR DESCRIPTION
**Pull request by isuruf**
Should only be merged after https://github.com/conda-forge/pocl-feedstock/pull/28 so that pocl.icd can be used.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -43,11 +36,9 @@ requirements:
         - six
         - mako
         - appdirs
-        - khronos-opencl-icd-loader  # [win]
+        - khronos-opencl-icd-loader  # [win or osx]
+        - ocl_icd_wrapper_apple  # [osx]
```
Comment (isuruf) [inline recipe/meta.yaml]: This library exposes Apple OpenCL implementation as an ICD.

**Comment by inducer** [comment]
[conda-forge/pocl-feedstock#28](https://github.com/conda-forge/pocl-feedstock/pull/28) is built.

This looks great. Thanks for working on this!

**Comment by inducer** [comment]
I'll update the PyOpenCL and pytential install instructions once this is built.

**Comment by inducer** [comment]
Uh-oh. Just realized that the pytential and pyopencl CIs will also need fixing.

**Pull request merged**